### PR TITLE
Update arithmeticController.js

### PR DESCRIPTION
**Pull request by skhalife**

### CODE REVIEW

**Review by Hollywood — CHANGES_REQUESTED** [review]
Hey, your tests are failing. It looks like they're concatenating strings instead of performing arithmetic.

File: api/controllers/arithmeticController.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ exports.calculate = function(req, res) {
   });
 
   var operations = {
-    'add':      function(a,b) { return +a + +b },
+    'add':      function(a,b) { return a + b },
```
Comment (Hollywood) [inline api/controllers/arithmeticController.js:14]: Add the + sign back to this line's return vars. JS will concatenate them as strings if they are not present.

**Review by Hollywood — CHANGES_REQUESTED** [review]

File: api/controllers/arithmeticController.js (hunk 1 above)
Comment (skhalife) [reply api/controllers/arithmeticController.js:14]: Thanks for adding. Javascript string is new to me!

Comment (Hollywood) [reply api/controllers/arithmeticController.js]: That's ok!! ✨

File: api/controllers/arithmeticController.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ exports.calculate = function(req, res) {
   });
 
   var operations = {
-    'add':      function(a,b) { return a + b },
+    'add':      function(a,b) { return +a + +b },
```
Comment (Hollywood) [inline api/controllers/arithmeticController.js]: This looks great!! :shipit:

**Review by Hollywood — APPROVED** [review]

**Pull request merged**